src/Plugin/Field/FieldType/OgMembershipReferenceItemList.php

```diff
@@ -75,15 +76,15 @@ public function postSave($update) {
     // todo: move to API function.
     $membership_ids = \Drupal::entityQuery('og_membership')
       ->condition('uid', $this->getEntity()->id())
-      ->condition('entity_type', $this->getFieldDefinition()->getTargetEntityTypeId())
+      ->condition('entity_type', $this->getFieldDefinition()->getFieldStorageDefinition()->getSetting('target_type'))
```

Comment:
Oh dear another one were this was mixed up! We already fixed a number of these. It would be a good idea to grep the entire code base to see if there are any more lingering around.